examples/bentoml_deployment/steps/predictor.py

```diff
@@ -33,7 +33,11 @@ def predictor(
     """
     service.start(timeout=10)  # should be a NOP if already started
     for img, data in inference_data.items():
-        prediction = service.predict("predict_ndarray", np.array(data))
+        grayscale_image = np.dot(
+            np.array(data)[..., :3], [0.2989, 0.5870, 0.1140]
+        )
+        grayscale_image = grayscale_image.astype(np.uint8)
+        prediction = service.predict("predict_ndarray", grayscale_image)
```

Comment:
why can't it predict on the original data directly anymore?

Reply:
Actually, I was surprised by how it was working from the start! the image we were using for inference where (28, 28, 3), and in the bentoml service validation we were asserting that it was greyscale (28, 28) and it was working somehow

Reply:
Ah so this is turning it to greyscale? In that case, just add a docstring please so we will remember this in the future 😄 

```
       grayscale_image = np.dot(
            np.array(data)[..., :3], [0.2989, 0.5870, 0.1140]
        )
```

Reply:
@safoinme will you add this?